addons/account/models/chart_template.py

```diff
@@ -516,9 +516,9 @@ def _post_load_data(self, template_code, company, template_data):
 
         # Set newly created journals as defaults for the company
         if not company.tax_cash_basis_journal_id:
```

Comment:
we might add `and company.tax_exigibility` in the condition :)

Reply:
Just adding that condition should solve the problem ?

Reply:
I think adding it doesn't resolve the problem

Reply:
of course it doesn't, it's just that it doesn't make sense to fill `tax_cash_basis_journal_id` if the company doesn't do CABA :)